package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "document-management-system",
+  "version": "1.0.0",
+  "description": "This repo contains an implementation of a full-stack document management system",
+  "main": "server.js",
+  "scripts": {
+    "test": "npm-run-all seed:test test:server reset:test",
+    "start": "babel-node server.js",
+    "e2e": "nightwatch",
+    "test:client": "jest",
+    "test:client:watch": "npm test -- --watch",
+    "db:migrate": "sequelize db:migrate",
+    "reset:test": "NODE_ENV=test babel-node server/test/TestHelpers/ResetDatabase.js",
+    "seed:test": "NODE_ENV=test babel-node server/test/TestHelpers/SeedDatabaseData.js",
+    "test:server": "NODE_ENV=test babel-node ./node_modules/babel-istanbul/lib/cli.js cover ./node_modules/.bin/_mocha server/test/**/*.spec.js && gulp coveralls",
+    "build:dist": "rimraf dist/ && babel ./ --out-dir dist/ --ignore ./node_modules,./server/test,./client/test,./reports,./migrations,./coverage,docs --copy-files",
+    "document": "rimraf docs/ && jsdoc -r -c ./conf.json --readme ./documentation/README.md --destination docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/andela-tAdedotun/document-management-system.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/andela-tAdedotun/document-management-system/issues"
+  },
+  "homepage": "https://github.com/andela-tAdedotun/document-management-system#readme",
+  "dependencies": {
+    "autoprefixer": "^7.1.1",
+    "axios": "^0.12.0",
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.8.0",
+    "babel-istanbul": "^0.12.2",
+    "babel-loader": "^6.2.4",
+    "babel-plugin-react-display-name": "^2.0.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-polyfill": "^6.8.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "6.5.0",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-register": "^6.8.0",
+    "bcrypt": "^1.0.2",
+    "body-parser": "^1.17.1",
+    "browser-sync": "^2.18.8",
+    "chai": "^3.5.0",
+    "classnames": "^2.2.5",
+    "colors": "^1.1.2",
+    "compression": "^1.6.1",
+    "coveralls": "^2.13.0",
```

Comment:
Countercheck the dependencies, `coveralls`, for example, and `react-hot-loader` should be in the devDependencies.